deploy/packages/bbp-packages.yaml

```
@@ -44,6 +44,7 @@ packages:
       #- python
     specs:
       - parquet-converters@0.3
+      - placement-algorithm@2.0.0
```

```suggestion
      - py-placement-algorithm@2.0.0
```

If this is not used as a library elsewhere but "end-user" software, I would recommend dropping the `py-` prefix from this package globally.

---

It's primary usage is a set of scripts; but it can be used as a library as well (that's why `app` variant in the recipe).
Does the syntax support aliasing (i.e. naming module differently than package?). If not, I'll follow your suggestion to rename the package.

---

I'm not too familiar with the module creation logic (on that front anyways), I think it's simpler to just rename the package. Note that you will still be able to use it as a library, since you derive from `PythonPackage`.